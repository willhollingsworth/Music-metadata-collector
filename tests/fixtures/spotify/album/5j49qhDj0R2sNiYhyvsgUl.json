{
  "album_type": "single",
  "total_tracks": 2,
  "available_markets": [
    "AR",
    "AU",
    "AT",
    "BE",
    "BO",
    "BR",
    "BG",
    "CA",
    "CL",
    "CO",
    "CR",
    "CY",
    "CZ",
    "DK",
    "DO",
    "DE",
    "EC",
    "EE",
    "SV",
    "FI",
    "FR",
    "GR",
    "GT",
    "HN",
    "HK",
    "HU",
    "IS",
    "IE",
    "IT",
    "LV",
    "LT",
    "LU",
    "MY",
    "MT",
    "MX",
    "NL",
    "NZ",
    "NI",
    "NO",
    "PA",
    "PY",
    "PE",
    "PH",
    "PL",
    "PT",
    "SG",
    "SK",
    "ES",
    "SE",
    "CH",
    "TW",
    "TR",
    "UY",
    "US",
    "GB",
    "AD",
    "LI",
    "MC",
    "ID",
    "JP",
    "TH",
    "VN",
    "RO",
    "IL",
    "ZA",
    "SA",
    "AE",
    "BH",
    "QA",
    "OM",
    "KW",
    "EG",
    "MA",
    "DZ",
    "TN",
    "LB",
    "JO",
    "PS",
    "IN",
    "BY",
    "KZ",
    "MD",
    "UA",
    "AL",
    "BA",
    "HR",
    "ME",
    "MK",
    "RS",
    "SI",
    "KR",
    "BD",
    "PK",
    "LK",
    "GH",
    "KE",
    "NG",
    "TZ",
    "UG",
    "AG",
    "AM",
    "BS",
    "BB",
    "BZ",
    "BT",
    "BW",
    "BF",
    "CV",
    "CW",
    "DM",
    "FJ",
    "GM",
    "GE",
    "GD",
    "GW",
    "GY",
    "HT",
    "JM",
    "KI",
    "LS",
    "LR",
    "MW",
    "MV",
    "ML",
    "MH",
    "FM",
    "NA",
    "NR",
    "NE",
    "PW",
    "PG",
    "PR",
    "WS",
    "SM",
    "ST",
    "SN",
    "SC",
    "SL",
    "SB",
    "KN",
    "LC",
    "VC",
    "SR",
    "TL",
    "TO",
    "TT",
    "TV",
    "VU",
    "AZ",
    "BN",
    "BI",
    "KH",
    "CM",
    "TD",
    "KM",
    "GQ",
    "SZ",
    "GA",
    "GN",
    "KG",
    "LA",
    "MO",
    "MR",
    "MN",
    "NP",
    "RW",
    "TG",
    "UZ",
    "ZW",
    "BJ",
    "MG",
    "MU",
    "MZ",
    "AO",
    "CI",
    "DJ",
    "ZM",
    "CD",
    "CG",
    "IQ",
    "LY",
    "TJ",
    "VE",
    "ET",
    "XK"
  ],
  "external_urls": {
    "spotify": "https://open.spotify.com/album/5j49qhDj0R2sNiYhyvsgUl"
  },
  "href": "https://api.spotify.com/v1/albums/5j49qhDj0R2sNiYhyvsgUl",
  "id": "5j49qhDj0R2sNiYhyvsgUl",
  "images": [
    {
      "url": "https://i.scdn.co/image/ab67616d0000b27332ed61b0e349694882fd0236",
      "height": 640,
      "width": 640
    },
    {
      "url": "https://i.scdn.co/image/ab67616d00001e0232ed61b0e349694882fd0236",
      "height": 300,
      "width": 300
    },
    {
      "url": "https://i.scdn.co/image/ab67616d0000485132ed61b0e349694882fd0236",
      "height": 64,
      "width": 64
    }
  ],
  "name": "Destroying the Commons (The Noam Chomsky Music Project)",
  "release_date": "2021-11-05",
  "release_date_precision": "day",
  "type": "album",
  "uri": "spotify:album:5j49qhDj0R2sNiYhyvsgUl",
  "artists": [
    {
      "external_urls": {
        "spotify": "https://open.spotify.com/artist/6ob0DwL4uI6eR5i9cc5tNn"
      },
      "href": "https://api.spotify.com/v1/artists/6ob0DwL4uI6eR5i9cc5tNn",
      "id": "6ob0DwL4uI6eR5i9cc5tNn",
      "name": "widerberg",
      "type": "artist",
      "uri": "spotify:artist:6ob0DwL4uI6eR5i9cc5tNn"
    },
    {
      "external_urls": {
        "spotify": "https://open.spotify.com/artist/7miGJahGF5JYOunptmXsQz"
      },
      "href": "https://api.spotify.com/v1/artists/7miGJahGF5JYOunptmXsQz",
      "id": "7miGJahGF5JYOunptmXsQz",
      "name": "Ian Urbina",
      "type": "artist",
      "uri": "spotify:artist:7miGJahGF5JYOunptmXsQz"
    }
  ],
  "tracks": {
    "href": "https://api.spotify.com/v1/albums/5j49qhDj0R2sNiYhyvsgUl/tracks?offset=0&limit=50",
    "limit": 50,
    "next": null,
    "offset": 0,
    "previous": null,
    "total": 2,
    "items": [
      {
        "artists": [
          {
            "external_urls": {
              "spotify": "https://open.spotify.com/artist/6ob0DwL4uI6eR5i9cc5tNn"
            },
            "href": "https://api.spotify.com/v1/artists/6ob0DwL4uI6eR5i9cc5tNn",
            "id": "6ob0DwL4uI6eR5i9cc5tNn",
            "name": "widerberg",
            "type": "artist",
            "uri": "spotify:artist:6ob0DwL4uI6eR5i9cc5tNn"
          },
          {
            "external_urls": {
              "spotify": "https://open.spotify.com/artist/7miGJahGF5JYOunptmXsQz"
            },
            "href": "https://api.spotify.com/v1/artists/7miGJahGF5JYOunptmXsQz",
            "id": "7miGJahGF5JYOunptmXsQz",
            "name": "Ian Urbina",
            "type": "artist",
            "uri": "spotify:artist:7miGJahGF5JYOunptmXsQz"
          }
        ],
        "available_markets": [
          "AR",
          "AU",
          "AT",
          "BE",
          "BO",
          "BR",
          "BG",
          "CA",
          "CL",
          "CO",
          "CR",
          "CY",
          "CZ",
          "DK",
          "DO",
          "DE",
          "EC",
          "EE",
          "SV",
          "FI",
          "FR",
          "GR",
          "GT",
          "HN",
          "HK",
          "HU",
          "IS",
          "IE",
          "IT",
          "LV",
          "LT",
          "LU",
          "MY",
          "MT",
          "MX",
          "NL",
          "NZ",
          "NI",
          "NO",
          "PA",
          "PY",
          "PE",
          "PH",
          "PL",
          "PT",
          "SG",
          "SK",
          "ES",
          "SE",
          "CH",
          "TW",
          "TR",
          "UY",
          "US",
          "GB",
          "AD",
          "LI",
          "MC",
          "ID",
          "JP",
          "TH",
          "VN",
          "RO",
          "IL",
          "ZA",
          "SA",
          "AE",
          "BH",
          "QA",
          "OM",
          "KW",
          "EG",
          "MA",
          "DZ",
          "TN",
          "LB",
          "JO",
          "PS",
          "IN",
          "BY",
          "KZ",
          "MD",
          "UA",
          "AL",
          "BA",
          "HR",
          "ME",
          "MK",
          "RS",
          "SI",
          "KR",
          "BD",
          "PK",
          "LK",
          "GH",
          "KE",
          "NG",
          "TZ",
          "UG",
          "AG",
          "AM",
          "BS",
          "BB",
          "BZ",
          "BT",
          "BW",
          "BF",
          "CV",
          "CW",
          "DM",
          "FJ",
          "GM",
          "GE",
          "GD",
          "GW",
          "GY",
          "HT",
          "JM",
          "KI",
          "LS",
          "LR",
          "MW",
          "MV",
          "ML",
          "MH",
          "FM",
          "NA",
          "NR",
          "NE",
          "PW",
          "PG",
          "PR",
          "WS",
          "SM",
          "ST",
          "SN",
          "SC",
          "SL",
          "SB",
          "KN",
          "LC",
          "VC",
          "SR",
          "TL",
          "TO",
          "TT",
          "TV",
          "VU",
          "AZ",
          "BN",
          "BI",
          "KH",
          "CM",
          "TD",
          "KM",
          "GQ",
          "SZ",
          "GA",
          "GN",
          "KG",
          "LA",
          "MO",
          "MR",
          "MN",
          "NP",
          "RW",
          "TG",
          "UZ",
          "ZW",
          "BJ",
          "MG",
          "MU",
          "MZ",
          "AO",
          "CI",
          "DJ",
          "ZM",
          "CD",
          "CG",
          "IQ",
          "LY",
          "TJ",
          "VE",
          "ET",
          "XK"
        ],
        "disc_number": 1,
        "duration_ms": 146147,
        "explicit": false,
        "external_urls": {
          "spotify": "https://open.spotify.com/track/2Yt0g0JZ3ymeRDthG04tQo"
        },
        "href": "https://api.spotify.com/v1/tracks/2Yt0g0JZ3ymeRDthG04tQo",
        "id": "2Yt0g0JZ3ymeRDthG04tQo",
        "name": "Destroying the Commons",
        "preview_url": null,
        "track_number": 1,
        "type": "track",
        "uri": "spotify:track:2Yt0g0JZ3ymeRDthG04tQo",
        "is_local": false
      },
      {
        "artists": [
          {
            "external_urls": {
              "spotify": "https://open.spotify.com/artist/6ob0DwL4uI6eR5i9cc5tNn"
            },
            "href": "https://api.spotify.com/v1/artists/6ob0DwL4uI6eR5i9cc5tNn",
            "id": "6ob0DwL4uI6eR5i9cc5tNn",
            "name": "widerberg",
            "type": "artist",
            "uri": "spotify:artist:6ob0DwL4uI6eR5i9cc5tNn"
          },
          {
            "external_urls": {
              "spotify": "https://open.spotify.com/artist/7miGJahGF5JYOunptmXsQz"
            },
            "href": "https://api.spotify.com/v1/artists/7miGJahGF5JYOunptmXsQz",
            "id": "7miGJahGF5JYOunptmXsQz",
            "name": "Ian Urbina",
            "type": "artist",
            "uri": "spotify:artist:7miGJahGF5JYOunptmXsQz"
          }
        ],
        "available_markets": [
          "AR",
          "AU",
          "AT",
          "BE",
          "BO",
          "BR",
          "BG",
          "CA",
          "CL",
          "CO",
          "CR",
          "CY",
          "CZ",
          "DK",
          "DO",
          "DE",
          "EC",
          "EE",
          "SV",
          "FI",
          "FR",
          "GR",
          "GT",
          "HN",
          "HK",
          "HU",
          "IS",
          "IE",
          "IT",
          "LV",
          "LT",
          "LU",
          "MY",
          "MT",
          "MX",
          "NL",
          "NZ",
          "NI",
          "NO",
          "PA",
          "PY",
          "PE",
          "PH",
          "PL",
          "PT",
          "SG",
          "SK",
          "ES",
          "SE",
          "CH",
          "TW",
          "TR",
          "UY",
          "US",
          "GB",
          "AD",
          "LI",
          "MC",
          "ID",
          "JP",
          "TH",
          "VN",
          "RO",
          "IL",
          "ZA",
          "SA",
          "AE",
          "BH",
          "QA",
          "OM",
          "KW",
          "EG",
          "MA",
          "DZ",
          "TN",
          "LB",
          "JO",
          "PS",
          "IN",
          "BY",
          "KZ",
          "MD",
          "UA",
          "AL",
          "BA",
          "HR",
          "ME",
          "MK",
          "RS",
          "SI",
          "KR",
          "BD",
          "PK",
          "LK",
          "GH",
          "KE",
          "NG",
          "TZ",
          "UG",
          "AG",
          "AM",
          "BS",
          "BB",
          "BZ",
          "BT",
          "BW",
          "BF",
          "CV",
          "CW",
          "DM",
          "FJ",
          "GM",
          "GE",
          "GD",
          "GW",
          "GY",
          "HT",
          "JM",
          "KI",
          "LS",
          "LR",
          "MW",
          "MV",
          "ML",
          "MH",
          "FM",
          "NA",
          "NR",
          "NE",
          "PW",
          "PG",
          "PR",
          "WS",
          "SM",
          "ST",
          "SN",
          "SC",
          "SL",
          "SB",
          "KN",
          "LC",
          "VC",
          "SR",
          "TL",
          "TO",
          "TT",
          "TV",
          "VU",
          "AZ",
          "BN",
          "BI",
          "KH",
          "CM",
          "TD",
          "KM",
          "GQ",
          "SZ",
          "GA",
          "GN",
          "KG",
          "LA",
          "MO",
          "MR",
          "MN",
          "NP",
          "RW",
          "TG",
          "UZ",
          "ZW",
          "BJ",
          "MG",
          "MU",
          "MZ",
          "AO",
          "CI",
          "DJ",
          "ZM",
          "CD",
          "CG",
          "IQ",
          "LY",
          "TJ",
          "VE",
          "ET",
          "XK"
        ],
        "disc_number": 1,
        "duration_ms": 190000,
        "explicit": false,
        "external_urls": {
          "spotify": "https://open.spotify.com/track/1w0YqmsRPGnUhw5WPCjbwL"
        },
        "href": "https://api.spotify.com/v1/tracks/1w0YqmsRPGnUhw5WPCjbwL",
        "id": "1w0YqmsRPGnUhw5WPCjbwL",
        "name": "Biologically Unique",
        "preview_url": null,
        "track_number": 2,
        "type": "track",
        "uri": "spotify:track:1w0YqmsRPGnUhw5WPCjbwL",
        "is_local": false
      }
    ]
  },
  "copyrights": [
    {
      "text": "2022 The Outlaw Ocean LLC",
      "type": "C"
    },
    {
      "text": "2022 The Outlaw Ocean LLC",
      "type": "P"
    }
  ],
  "external_ids": {
    "upc": "196835404769"
  },
  "genres": [],
  "label": "The Outlaw Ocean LLC",
  "popularity": 38
}