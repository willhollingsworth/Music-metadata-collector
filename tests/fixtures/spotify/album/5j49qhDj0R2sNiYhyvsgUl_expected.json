{
  "album_name": "Destroying the Commons (The Noam Chomsky Music Project)",
  "album_id": "5j49qhDj0R2sNiYhyvsgUl",
  "popularity": 38,
  "artist_name": "widerberg",
  "artist_id": "6ob0DwL4uI6eR5i9cc5tNn",
  "release_date": "2021-11-05",
  "album_type": "single",
  "track_count": 2,
  "tracks": []
}